src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -2934,6 +2934,63 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
             }
         }
 
+        [ActiveIssue(31424)]
+        [OuterLoop("Uses external server")]
+        [Fact]
+        public async Task SendAsync_Http2RemoteServerWithCookie_Success()
```

Comment:
Most of the cookie tests for HttpClientHandler are here:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/tests/FunctionalTests/HttpCookieProtocolTests.cs

You should probably put additional cookie tests into that file.

Reply:
Agree, will move to `HttpCookieProtocolTests.cs`.